coverage.ignore.in

```diff
@@ -0,0 +1,12 @@
+profiler/src/Remotery/lib/*
+profiler/src/Remotery/vis/*
+graphics/src/tiny_obj_loader.h
+av/include/ignition/common/HWVideo.hh
+av/include/ignition/common/HWEncoder.hh
+av/src/HWEncoder.cc
+include/ignition/common/FlagSet.hh
+src/FilesystemBoost.cc
+src/win_dirent.h
+examples/assert_example.cc
+graphics/src/tinyxml2/tinyxml2.h
+graphics/src/tinyxml2/tinyxml2.cpp
```

Comment:
```suggestion
profiler/src/Remotery/*
graphics/src/tiny_obj_loader.h
src/win_dirent.h
graphics/src/tinyxml2/*
```

We should not add these to the ignore file:
- HW encoding files since they were implemented in https://github.com/gazebosim/gz-common/pull/125
- `FlagSet.hh` there is modified code and tests were added for this https://github.com/gazebosim/gz-common/pull/118
- `FilesystemBoost.cc` we modified this and it has high coverage already

Reply:
Add back in c36ceb0